pkg/avancement/service_manager.go

```diff
@@ -194,15 +215,15 @@ func encode(gitURL, branch string) string {
 	return url.QueryEscape(gitURL) + "-" + url.QueryEscape(branch)
 }
 
-func addCredentialsIfNecessary(s string, a *git.Author) (string, error) {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Good point actually, yeah it's obvious s is a string and an a is author indeed 😆 will change back and remember